Autopilot/Inc/xbee.hpp

```diff
@@ -0,0 +1,45 @@
+/**
+ * Xbee Sensor Functions
+ * Author: Gordon Fountain
+ */
+
+#ifndef XBEE_HPP
+#define XBEE_HPP
+
+#include "TelemPathInterface.hpp" //Holds structs for sent and received data
+
+//Ask Dhruv R (Elias) about UART stuff (Or ask our lord FREERTOS in the sky)
+
+class XBEE{
+    public:
+        /**
+         * Initializes XBEE
+         * */
+        virtual void Init() = 0; 
```

Comment:
This was a very early design for it, I stopped working on it when I got blocked on the actual xbees, not soo sure how this will actually be implemented.

Reply:
I would appreciate any suggestions you have